coverage.cfg

```diff
@@ -6,6 +6,7 @@ omit =
      aiosmtpd/testing/*
      aiosmtpd/tests/*
     .tox/*
+     */helpers/pycharm/*
```

Comment:
When using pytest in PyCharm, somehow coverage.py can "break out" of the virtualenv and start testing coverage of PyCharm's helper scripts. This prevents those scripts from figuring out in coverage.

Reply:
This has been brought over to `pyproject.toml`